src/renderer/components/_atoms/select/index.tsx

```diff
@@ -44,13 +44,15 @@ const SelectContent = ({
         side={side}
         {...res}
       >
```

Comment:
Later, this can be refactored. Using the composition pattern, we can make the scroll button a prop to be used, or not, by the father component.